### PR TITLE
Add version: BraveSoftware.BraveBrowser.Dev version 96.1.34.72

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38785)

### CODE REVIEW

File: manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser.Dev
+PackageVersion: 96.1.34.72
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.yaml:5]:
```suggestion
PackageVersion: 97.1.34.72
```
image

File: manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser.Dev
+PackageVersion: 96.1.34.72
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 97.1.34.72
```
image

File: manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser.Dev
+PackageVersion: 96.1.34.72
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.installer.yaml:5]:
```suggestion
PackageVersion: 97.1.34.72
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.installer.yaml (hunk 3 above)
Comment (OfficialEsco) [inline manifests/b/BraveSoftware/BraveBrowser/Dev/96.1.34.72/BraveSoftware.BraveBrowser.Dev.installer.yaml:5]:
```suggestion
PackageVersion: 97.1.34.72
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
@vedantmgoyal2009 Version needs to be bumped in automation

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**